TEST_APPS/device/nfcapp/nfcprocess.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2018 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _NFCPROCESS_H_INCLUDED
+#define _NFCPROCESS_H_INCLUDED
+
+#include <string>
+#include <vector>
+#include <stdio.h>
+#include <stdarg.h>
+
+#include "nfc/ndef/MessageBuilder.h"
+#include "nfc/ndef/common/URI.h"
+#include "nfc/ndef/common/util.h"
+#include "nfctestshim.h"
+
+#if MBED_CONF_NFCEEPROM
+    #include "NFCEEPROM.h"
+    #include "EEPROMDriver.h"
+#else
+	#include "nfc/nfcdefinitions.h"
```

Comment:
please fix indentation,
change your IDE to produce spaces and not tabs

Reply:
thanks, astyle tidied up now.